modules/xfem/include/materials/LevelSetBiMaterialStress.h

```diff
@@ -0,0 +1,49 @@
+/****************************************************************/
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*          All contents are licensed under LGPL V2.1           */
+/*             See LICENSE for full restrictions                */
+/****************************************************************/
+#ifndef LEVELSETBIMATERIALSTRESS_H
+#define LEVELSETBIMATERIALSTRESS_H
+
+#include "LevelSetBiMaterial.h"
+
+// Forward Declarations
+class LevelSetBiMaterialStress;
+class RankTwoTensor;
+class RankFourTensor;
+class XFEM;
+
+registerMooseObject("XFEMApp", LevelSetBiMaterialStress);
+
+template <>
+InputParameters validParams<LevelSetBiMaterialStress>();
+
+/**
+ * Compute the stress for bi-materials problem (consisting of two different materials) defined by a
+ * level set function
+ *
+ */
+class LevelSetBiMaterialStress : public LevelSetBiMaterial
+{
+public:
+  LevelSetBiMaterialStress(const InputParameters & parameters);
+
+protected:
+  virtual void assignQpPropertiesForLevelSetPositive();
+  virtual void assignQpPropertiesForLevelSetNegative();
+
+  /// bi-material stress
```

Comment:
I think I'd say: 'vector of stresses for the two separate materials in the bi-material system'